product-docs/installation/prometheus-metrics.md

```
@@ -0,0 +1,119 @@
+
+### Verifying the endpoint
+
+From a machine that has access to the backend (not from the public internet):
+
+```bash
+curl http://localhost:8000/metrics
+```
+
+You should see output similar to:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)